content/en/apps/reference/app-settings/transitions.md

```diff
@@ -381,6 +406,23 @@ Entries in `muting_history` contain the following information:
 | date | Date in ISO Format |
 | report_id | An `_id` reference to the report that triggered the action |
 
+
+##### [11] Client-side Muting history as of 3.12.0
+Each time the client changes the `muted` state of a contact, an entry is added to a `muting_history` property on the contact's doc. The `last_update` entry is also changed to `client`.
+The `muting_history` property contains the following information:
+
+| Property | Values | Description | 
+| --- | ---- | ---- |
+| last_update | `server` or `client` | Updated every time a service updates the contact, with the corresponding value | 
+| server | Object |  |
+| server.muted | `true` or `false` | Last known server muting state |
+| server.date | Date in ISO format | Last known server muting/unmuting date | 
+| client | Array | Client-side muting/unmuting events list. <br> New events are pushed at the end of this list and it should never be re-ordered. <br>The list represents the "chronological" order in which the reports that triggered muting were created. | 
```

Comment:
Just a note to update this based on the result of this conversation.

Reply:
Editing this would require quite a large refactor, I was just planning to edit the config.

Reply:
This part: https://github.com/medic/cht-docs/pull/504/files#diff-03618adbb0b596d7e314daf185764395f0b911b479b72ac9954722007b776c93R25

Reply:
Ah, I see. Would other potential future client-side transitions go here as well? Does having client here and client_side on the config make it more confusing? I'll let you make a call on that with app developers.

Reply:
It depends, right now this is an implementation detail for muting specifically, which is in itself a complex piece of work. It's used exclusively to have consistent muting while having out of order syncs. 
App developers or anyone really should be using these.

Reply:
But I'll do the change anyway :P